src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -203,6 +203,11 @@ public string HubName
         /// </summary>
         public bool UseAppLease { get; set; } = true;
 
+        /// <summary>
+        /// If IncludeParameters is true, the input of the activity function will be displayed.
```

Comment:
I suggest we update this comment. This setting works for both activities and sub-orchestrations. It's not clear what is meant when we say it "will be displayed", so I suggest wording that suggests that any tool can display this information.
```suggestion
        /// If <c>true</c>, the inputs of activity and sub-orchestrator functions will be saved to the orchestration history, making them visible to debugging tools. The default value is <c>false</c>.
```

Reply:
Thanks, Chris! I have already updated it.